test/addons/argocd/test

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+from drenv import kubectl
+from drenv import commands
+
+
+def wait(cluster):
+    print("Waiting application to be healthy")
+    kubectl.wait(
+        "application",
+        "guestbook",
+        "--for=jsonpath={.status.health.status}=Healthy",
+        "--namespace=argocd",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+
+if len(sys.argv) != 4:
+    print(f"Usage: {sys.argv[0]} hub cluster1 cluster2")
+    sys.exit(1)
+
+hub = sys.argv[1]
+
+commands.run(
+    "argocd",
+    "app",
+    "create",
+    "guestbook",
+    "--repo",
+    "https://github.com/argoproj/argocd-example-apps.git",
```

Comment:
Can we use local resources for the test app or we must use a git repo?

If we must use a git repo we can use a local git repo - we have automated creation of a local git repo: https://github.com/RamenDR/ramen/tree/main/test/gitlap.

Reply:
for argocd, it must use a git repo.  Did not understand how we could use a local git repo inside minikube in an automated way with cache...

Reply:
After configuring the local git repo based on:
https://github.com/RamenDR/ramen/tree/main/test/gitlap

You can access this repo within the cluster like we show here:
https://github.com/RamenDR/ramen/tree/main/test/gitlap#testing-the-server

So if we clone the remote repo and push it to the local git repo, we should be able to deploy the app using the local git repo instead of github using:

    --repo=http://host.minikube.internal/argocd-example-apps.git

I think we can open an issue for this work and do this later.